src/services/session.service.ts

```diff
@@ -23,7 +23,7 @@ export class SessionService {
     try {
       this.userService.hashUserPassword(user);
       return await this.userRepository.save(user);
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
could we try to not use `any`? Maybe `unknown` or custom type would be a better approach

Reply:
Yes, totally agree, I replaced within `unknown`

Reply:
As we discussed, I removed the type and throwed a custom exception without using any information of the error.